CalibTracker/SiStripHitEfficiency/plugins/HitEff.cc

```diff
@@ -267,7 +265,7 @@ void HitEff::analyze(const edm::Event& e, const edm::EventSetup& es) {
       surface = &(tracker->idToDet(ClusterDetId)->surface());
       LocalPoint lp = parameters.first;
       GlobalPoint gp = surface->toGlobal(lp);
-      unsigned int layer = checkLayer(ClusterId, tTopo);
+      unsigned int layer = ::checkLayer(ClusterId, tTopo);
             if(DEBUG) cout << "Found hit in cluster collection layer = " << layer << " with id = " << ClusterId << "   local X position = " << lp.x() << " +- " << sqrt(parameters.second.xx()) << "   matched/stereo/rphi = " << ((ClusterId & 0x3)==0) << "/" << ((ClusterId & 0x3)==1) << "/" << ((ClusterId & 0x3)==2) << endl;
```

Comment:
Should perhaps this cout be moved to LOGPRINT too?

Reply:
It can, bug it's protected by a flag (also this module is not run, and not meant to run in central production), so I'll update the file only if there are more substantial comments, otherwise I'd prefer to let it for a follow-up.

Reply:
done in the last push.